tensorflow/tools/ci_build/builds/pip.sh

```diff
@@ -119,7 +127,10 @@ echo "whl file path = ${WHL_PATH}"
 # Install, in user's local home folder
 echo "Installing pip whl file: ${WHL_PATH}"
 
+# Call pip install twice, first time with --upgrade and second time without it
+# This addresses the sporadic test failures related to protobuf version
 ${PYTHON_BIN_PATH} -m pip install -v --user --upgrade ${WHL_PATH} &&
+${PYTHON_BIN_PATH} -m pip install -v --user ${WHL_PATH} &&
```

Comment:
I know "set -e" is used in some other files in tensorflow/tools/ci_build. But it is considered unreliable (http://mywiki.wooledge.org/BashFAQ/105/Answers). This is why I prefer to check for error manually using && at critical places in the script.